### PR TITLE
feat: Button 

### PR DESCRIPTION
**Pull request by tushartiwari7**
## Types of Buttons included
- Primary Button
- Outlined Button
- Icon Button
-  Link Button
- Sized Buttons
- Rounded Buttons

### CODE REVIEW

**Review by sks-21 — APPROVED** [review]
The code looks nice. Good to merge.

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
+.btn-wrapper {
+  gap: 2rem;
+}
+
+.btn {
+  border: 0;
+  color: white;
+  cursor: pointer;
+  background: white;
```
Comment (JantuDeb) [inline components/button/button.css]: `background` and `color` values should not be the same.

**Review by JantuDeb — APPROVED** [review]
This looks good include the changes I have commented then you can merge.

File: components/button/button.css (hunk 1 above)
Comment (tushartiwari7) [reply components/button/button.css]: Thanks jantu for finding this vulnerability 💯

**Pull request merged**